cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/counters/MessageStatisticsCollector.java

```diff
@@ -16,10 +16,10 @@
 package com.exactpro.cradle.cassandra.counters;
 
 import com.exactpro.cradle.BookId;
-import com.exactpro.cradle.serialization.SerializedEntityMetadata;
+import com.exactpro.cradle.serialization.SerializedMessageMetadata;
 
 import java.util.Collection;
 
 public interface MessageStatisticsCollector {
-    void updateMessageBatchStatistics(BookId bookId, String page, String sessionAlias, String direction, Collection<SerializedEntityMetadata> batchMetadata);
+    void updateMessageBatchStatistics(BookId bookId, String page, String sessionAlias, String direction, Collection<SerializedMessageMetadata> batchMetadata);
```

Comment:
I think we should be consistent here
```suggestion
    void updateMessageBatchStatistics(BookId bookId, String page, String sessionAlias, String direction, Collection<? extends SerializedMessageMetadata> batchMetadata);
```

Reply:
There returns specific implementation instead of SerializedEntityMetadata